app/controllers/friends/new.js

```diff
@@ -2,9 +2,8 @@ import FriendsBaseController from './base';
 
 export default FriendsBaseController.extend({
   actions: {
-    cancel: function() {
+    cancel() {
```

Comment:
'concise methods' is available in ES6 (use esnext option) or Mozilla JS extensions (use moz).